src/AdminGuesser.test.tsx

```diff
@@ -83,4 +86,36 @@ describe('<AdminGuesser />', () => {
 
     expect(renderer.getRenderOutput()).toMatchSnapshot();
   });
+
+  test('renders with admin element', () => {
+    const authProvider: AuthProvider = {
+      getPermissions: () => Promise.resolve(['user']),
+      getIdentity: () =>
+        Promise.resolve({
+          id: '/users/2',
+          fullName: 'Test User',
+          avatar: undefined,
+        }),
+      login: () => Promise.resolve(),
+      logout: () => Promise.resolve(),
+      checkAuth: () => Promise.resolve(),
+      checkError: () => Promise.resolve(),
+    };
+
+    const AdminEl = (props: AdminProps) => (
+      <AuthContext.Provider value={authProvider}>
+        <AdminUI {...props} />
+      </AuthContext.Provider>
+    );
+
+    renderer.render(
+      <AdminGuesser
+        admin={AdminEl}
+        dataProvider={dataProvider}
+        schemaAnalyzer={schemaAnalyzer()}
+      />,
+    );
+
+    expect(renderer.getRenderOutput()).toMatchSnapshot();
```

Comment:
Maybe matching the snapshot is too much.

Reply:
Indeed, I'll change it.